eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Ledgers.hs

```diff
@@ -62,6 +66,27 @@ data ShelleyLedgersEnv era = LedgersEnv
   , ledgersPp :: PParams era
   , ledgersAccount :: AccountState
   }
+  deriving (Generic)
+
+deriving instance Eq (PParamsHKD Identity era) => Eq (ShelleyLedgersEnv era)
+
+deriving instance Show (PParamsHKD Identity era) => Show (ShelleyLedgersEnv era)
+
+instance NFData (PParamsHKD Identity era) => NFData (ShelleyLedgersEnv era)
+
+instance
```

Comment:
You could mention the  `Generic`, `Eq`, `Show`,  `NFData` and `EncCBOR` instances for `ShelleyLedgersEnv` in the shelley CHANGELOG

Reply:
Done